tools/run_tests/artifacts/build_artifact_python.sh

```diff
@@ -22,6 +22,9 @@ export PYTHON=${PYTHON:-python}
 export PIP=${PIP:-pip}
```

Comment:
Is there an equivalent Windows build that should be updated as well?

Reply:
Yes, I have done it in #21340. I assumed Linux would be in good shape, but no.